### DISCUSSION TITLE
逆向绳网

### DISCUSSION THREAD

**Discussion by muxueano** [share121/inter-knot#4531] [教程]
![1000246876](https://github.com/user-attachments/assets/67569eeb-7115-4a54-8a87-92ad8527e616)
修改绳网APP图片或字体
必备软件（MT管理器）
玩过MT管理器的就看7
教程
1.同意权限
2.同意权限后“点击左上角三条杠”
3.下滑找到“安装包提取”
4.搜索绳网
5.点击绳网并“提取安装包后点击定位”
6.点击绳网并查看
7.按路径一次点击（/assets/flutter_assets/assets/images/）
可修改文件[文件名字必须一模一“这里使用{｜}区分可修改的文件（default-cover.webp｜profile-photo.webp｜default-cover.webp）]
注:修改ttf文件会导致图标显示不了

_Originally posted by @muxueano in https://github.com/share121/inter-knot/discussions/4194_

**Comment by link-fgfgui** [comment]
改别人app是不礼貌的,礼貌的方式是fork仓库(~顺手点个star~)改源码并重新编译
你这个标题完全表达不出你的主要观点,个人建议改掉,改成`[教程]修改背景图`这种会好一些
